src/client/components/create/CreateGameForm.vue

```
@@ -659,6 +660,12 @@ export default (Vue as WithRefs<Refs>).extend({
         const readerResults = reader.result;
         if (typeof(readerResults) === 'string') {
           const results = JSON.parse(readerResults);
+
+          const source = results['source'];
+          if(source !== window.location.origin) {
+            alert($t('Source of uploaded settings is different, there might be some inconsistencies'));
```

This is good work, but as I think about it, the feature, as written on Github Issues probably hurts more than it helps. Here's why:

1. What's really important is not the source, but  that the parameters make sense. Right now the app will silently swallow any settings without checking that they make sense, and without warning somebody creates a game that doesn't have the parameters they expect.
2. If we change download/upload settings, the domain isn't really enough to validate that the settings should match.
3. If we change download/upload settings, the domain isn't really enough to validate that the settings should match.
4. As-is, this is going to immediately block any player that uploads settings; so an alert will be more disruptive than helpful. This code could special-case when source is empty, but that also doesn't help.

Sorry for this one. I thought it was a good starter, but now I think not. :/ Feel free to DM on discord if you would like to pick up another change. (The one specified in #1, above, might even be good.)

---

I am closing the issue. If you want to do something else, all you have to do is respond.